src/runtime/getPositions.ts

```diff
@@ -392,17 +393,23 @@ function addSourceAppId<T>(definition: T, sourceAppId: string) {
 }
 
 // This is the main logic to get positions
-export async function getPositions(
-  networkId: NetworkId,
-  address: string | undefined,
-  appIds: string[] = [],
-) {
+export async function getPositions({
+  networkId,
+  address,
+  appIds = [],
+  t,
+}: {
+  networkId: NetworkId
+  address: string | undefined
+  appIds: string[]
+  t?: TFunction<'translation', undefined>
```

Comment:
Also wanted to make it required here, but again a lot of tests use it and wasn't sure how to mock well.

Reply:
ah yes, I propose we let actual i18n happen.
See https://github.com/valora-inc/hooks/pull/505